apps/are_form/lib/classARE.js

```diff
@@ -1,143 +1,147 @@
+var staticAppealStages = require('../lib/staticAppealStages');
 
 module.exports.Calculator = class {
 
     constructor(date, country, appealstage) {
-        this.baseDate = moment(date,dateformat).format(dateformat);
+        this.baseDate = moment(date, dateformat).format(dateformat);
         this.country = country;
         this.appealStage = appealstage;
         this.excludedDates = [];
-        this.isBaseWeekend      = this.isWeekend(this.baseDate);
+        this.isBaseWeekend = this.isWeekend(this.baseDate);
         this.isBaseExclusionDay = this.isExclusionDay(this.baseDate);
-        this.startDate          = moment(this.setStartDate(this.getExclusionDates()),dateformat).format(dateformat);
-        this.appealInfo         = this.getAppealInfo(this.appealStage);
-        this.areDate            = moment(this.calculateAREDate(this.appealInfo),dateformat).format(dateformat);
-        this.excludedDateRange = moment(staticExclusionDates.getFirstExclusionDate(),"YYYY-MM-DD").format(dateformat) +
-                      ' to '  + moment(staticExclusionDates.getLastExclusionDate(),"YYYY-MM-DD").format(dateformat);
+        this.startDate = moment(this.setStartDate(this.getExclusionDates()), dateformat).format(dateformat);
+        this.appealInfo = this.getAppealInfo(this.appealStage);
+        this.areDate = moment(this.calculateAREDate(this.appealInfo), dateformat).format(dateformat);
+        this.excludedDateRange = moment(staticExclusionDates.getFirstExclusionDate(), 'YYYY-MM-DD').format(dateformat) +
+                      ' to ' + moment(staticExclusionDates.getLastExclusionDate(), 'YYYY-MM-DD').format(dateformat);
         this.baseBeforeEarliestExclusionDate = this.isBaseBeforeEarliestExclusion();
         this.areAfterLastExclusionDate = this.isAREAfterLastExclusion();
-
-    };
+    }
 
     addDays(toDate, daysToAdd) {
-      return moment(toDate, dateformat).add(daysToAdd, 'days');
-    };
+        return moment(toDate, dateformat).add(daysToAdd, 'days');
+    }
 
     addMonths(toDate, monthsToAdd) {
-      return moment(toDate, dateformat).add(monthsToAdd, 'months');
-    };
+        return moment(toDate, dateformat).add(monthsToAdd, 'months');
+    }
 
     addDaysIgnoringWeekendsAndExclusionDays(toDate, daysToAdd, exclusionDays) {
-  	    var count = 0 ;
+        var count = 0;
         var tempDate = moment(toDate, dateformat);
-  	    while (count < daysToAdd ) {
-          tempDate = this.addDays(tempDate, 1)
-
-  				if (this.isWeekend(tempDate) === false &&
-              this.isExclusionDay(tempDate) === false) {
-  		   		count++
-  				}
-  	    }
-  	    return tempDate;
-  	};
+        while (count < daysToAdd) {
+            tempDate = this.addDays(tempDate, 1);
+            if (this.isWeekend(tempDate) === false &&
+                this.isExclusionDay(tempDate) === false) {
+                count++;
+            }
+        }
+        return tempDate;
+    }
 
+    /* eslint complexity: 1 */
```

Comment:
This might be better to use `/* eslint-disable complexity */` and then `/* eslint-enable complexity */` once you're past the bad function. I think here you're disabling the complexity check for the rest of the file (unless that's what you want?)